Sources/ClientRuntime/Retries/RetryError.swift

```diff
@@ -14,7 +14,7 @@ public enum RetryError {
 }
 
 public extension RetryError {
-    func toCRTType() -> CRTRetryError {
+    func toCRTType() -> AwsCommonRuntimeKit.RetryError {
```

Comment:
isn't is `CommonRuntimeKit` now?

Reply:
Negative, [CommonRuntimeKit](https://github.com/awslabs/aws-crt-swift/blob/f3ae10c9a33bdc9d04246141ef0eb5840b369205/Source/AwsCommonRuntimeKit/CommonRuntimeKit.swift) is a struct while `AwsCommonRuntimeKit` is the module. 

We should clean up these names though. It's odd to have something with a `Kit` suffix that isn't a module.